.github/workflows/build-docker.yaml

```diff
@@ -1,48 +1,58 @@
-name: Build scpca-tools image
+name: Build scpca-tools images
 
 on:
   push:
     branches:
       - main
     tags:
-      - 'v*'
+      - "v*"
   pull_request:
     branches:
       - main
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: alexslemonade/scpca-tools
+  IMAGE_NAME: alexslemonade/scpcatools
```

Comment:
I'm fine with making this change everywhere. We will just need to note in https://github.com/AlexsLemonade/scpca-nf/issues/747 to also change the image name.